### PR TITLE
Documentation Openstack

### PR DESCRIPTION
**Pull request by lapentad**
Tested the ability of installing Nephio on multiple Openstack clusters, using the test-infra's ansible script.
[Used this flow](https://www.kubermatic.com/blog/multi-cluster-deployment-management-with-nephio-a-first-guide/) to sync the clusters and deploy packages via Github.

### CODE REVIEW

File: install-guide/openstack.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# Openstack Multi-Cluster Deployment Management with Nephio
```
Comment (electrocucaracha) [inline install-guide/openstack.md:1]: nit:
```suggestion
# OpenStack Multi-Cluster Deployment Management with Nephio

```

File: install-guide/openstack.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    name: data-gitea-0
+    spec:
+    storageClassName: standard
+    accessModes:
+        - ReadWriteOnce
+    resources:
+        requests:
+        storage: 10Gi
+namespace: "{{ gitea.k8s.namespace }}"
+```
```
Comment (electrocucaracha) [inline install-guide/openstack.md:94]: Maybe those resources can add to the [gitea package](https://github.com/nephio-project/nephio-example-packages/tree/main/gitea) to simplify the process

File: install-guide/openstack.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  2. Change context value from all the *test-infra\e2e\provision\playbooks* yaml files into your kubernetes context: `kubectl config get-contexts`
+    
+    context: kubernetes-admin@cluster.local
```
Comment (electrocucaracha) [inline install-guide/openstack.md:97]: This change needs to be done in the `test-infra` project, there are some places using `k8s.context` var like this [one](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/roles/bootstrap/defaults/main.yml#L22) but there are others using hardcode like [here](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/cluster.yml#L112)

File: install-guide/openstack.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+```
+  2. Change context value from all the *test-infra\e2e\provision\playbooks* yaml files into your kubernetes context: `kubectl config get-contexts`
+    
+    context: kubernetes-admin@cluster.local 
+  
+  3. Disable kind installation in 
+  *test-infra\e2e\provision\playbooks\roles\bootstrap\defaults\main.yml*
+   
+    kind:
+    enabled: false
```
Comment (electrocucaracha) [inline install-guide/openstack.md:103]:
```suggestion
    kind:
      enabled: false

```

File: install-guide/openstack.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+  *test-infra\e2e\provision\playbooks\roles\bootstrap\defaults\main.yml*
+   
+    host_min_vcpu: 4 
+    host_min_cpu_ram: 8
+
+  5. Run the installation script in
+  *test-infra\e2e\provision\install_sandbox.sh*
+
+## Manual Installation of the managment cluster using kpt
+TDB (manual install of kpt, porch, configsync, nephio-webui, capi, metallb)
```
Comment (electrocucaracha) [inline install-guide/openstack.md:115]: Ideally the management cluster should be installed with the [Install Ansible role](https://github.com/nephio-project/test-infra/tree/main/e2e/provision/playbooks/roles/install)

File: install-guide/openstack.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+
+## Manual Installation of the Edge cluster using kpt
```
Comment (electrocucaracha) [inline install-guide/openstack.md:117]: I think that edge clusters should be provisioned by [Cluster API](https://github.com/kubernetes-sigs/cluster-api-provider-openstack), which means that we need to create a new package on the existing catalog like this [one](https://github.com/nephio-project/nephio-example-packages/tree/main/cluster-capi-infrastructure-docker) or something new [here](https://github.com/nephio-project/catalog/tree/main/infra/capi)

**Review by electrocucaracha — COMMENTED** [review]
Glad that you started documenting this, I consider that this scenario must be cover, hopefully we can simplify the process to better adoption.

File: install-guide/openstack.md (hunk 3 above)
Comment (lapentad) [reply install-guide/openstack.md:98]: Yes I manually changed all of them. I know this is not ideal, but I wanted to see if the ansible script would also work in a different environment.

File: install-guide/openstack.md (hunk 6 above)
Comment (lapentad) [reply install-guide/openstack.md:116]: Hi Victor, I wanted to address the use case where the user doesn't have access to the provisioning process. For instance, I do not have access to the cluster provisioning cli within my organization.

File: install-guide/openstack.md (hunk 3 above)
Comment (electrocucaracha) [reply install-guide/openstack.md:98]: I'll submit the PR for that and expose the value in a variable.

File: install-guide/openstack.md (hunk 6 above)
Comment (electrocucaracha) [reply install-guide/openstack.md:116]: Got it, maybe we need to specify that as requirement on the [prerequesites section](https://github.com/nephio-project/docs/blob/main/install-guide/byoc.md#prerequisites) or in this document.

\cc @johnbelamaric

**Comment by liamfallon** [comment]
I think this could be pulled in now

File: install-guide/openstack.md (hunk 3 above)
Comment (electrocucaracha) [reply install-guide/openstack.md:98]: @lapentad I have created a [PR](https://github.com/nephio-project/test-infra/pull/217) to simplify this process, once it's available you could override some default values with

```bash
ANSIBLE_CMD_EXTRA_VAR_LIST="k8s.context=kubernetes-admin@cluster.local,kind.enabled=false" ./install_sandbox.sh
```

**Pull request merged**